atuin-client/src/history.rs

```diff
@@ -24,6 +24,7 @@ pub struct History {
 // Forgive me, for I have sinned
 // I need to replace rmp with something that is more backwards-compatible.
 // Protobuf, or maybe just json
+#[allow(clippy::module_name_repetitions)]
```

Comment:
If there is a good idea for a new name, this would be an easy fix to rename the symbol...